test-cases/tests/installation/a46-verify-migration-to-obo-was-successful.md

```diff
@@ -1,3 +1,15 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+      - osd-post-upgrade
+    targets:
+      - 1.38.0
+    tags:
+      - manual-selection
```

Comment:
wrong indentation, `tags` must be on the same level as `products`. See e.g.
https://raw.githubusercontent.com/integr8ly/integreatly-operator/master/test-cases/tests/alerts/c11b-verify-rhoam-install-alert-is-firing.md